### PR TITLE
VAVFA-9061: Display verbiage for hours on caps lists.

### PR DESCRIPTION
**Pull request by ethanteague**
## Description
closes https://github.com/department-of-veterans-affairs/va.gov-cms/issues/9061

## Testing done
Visual / build

## Screenshots
image

## Acceptance criteria
- [ ] on local version of `content-build` running yarn preview with prod as endpoint, go to `preview?nodeId=28246` and visually verify cap location listings show `Veterans should call main Vet Center for hours ` verbiage below phone numbers

## Definition of done
- [ ] Events are logged appropriately
- [ ] Documentation has been updated, if applicable
- [ ] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [ ] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

File: src/site/includes/vet_centers/address_phone_image.liquid [hunk 1]
```diff
@@ -62,10 +62,14 @@
         <div class="vads-u-margin-bottom--3">
             {% include 'src/site/includes/hours.liquid' with allHours = vetCenter.fieldOfficeHours headerType = 'small' %}
         </div>
-      {% elsif vetCenter.entityBundle == "vet_center_cap" or vetCenter.entityBundle == "vet_center" or vetCenter.entityBundle == "vet_center_outstation" %}
+      {% elsif vetCenter.entityBundle == "vet_center_cap" and vetCenterUseCapHours == true or vetCenter.entityBundle == "vet_center" or vetCenter.entityBundle == "vet_center_outstation" %}
         <div class="vads-u-margin-bottom--3">
             {% include 'src/site/includes/hours.liquid' with allHours = vetCenterHours headerType = 'small' %}
         </div>
+      {% elsif vetCenter.entityBundle == "vet_center_cap" and vetCenterUseCapHours == false %}
```
Comment (ndouglas) [inline src/site/includes/vet_centers/address_phone_image.liquid:69]: Seems redundant with above addition.

File: src/site/includes/vet_centers/address_phone_image.liquid [hunk 2]
```diff
@@ -62,10 +62,14 @@
         <div class="vads-u-margin-bottom--3">
             {% include 'src/site/includes/hours.liquid' with allHours = vetCenter.fieldOfficeHours headerType = 'small' %}
         </div>
-      {% elsif vetCenter.entityBundle == "vet_center_cap" or vetCenter.entityBundle == "vet_center" or vetCenter.entityBundle == "vet_center_outstation" %}
+      {% elsif vetCenter.entityBundle == "vet_center_cap" and vetCenterUseCapHours == true or vetCenter.entityBundle == "vet_center" or vetCenter.entityBundle == "vet_center_outstation" %}
```
Comment (ndouglas) [inline src/site/includes/vet_centers/address_phone_image.liquid:65]: What's the operator precedence here? 🤔

File: src/site/includes/vet_centers/address_phone_image.liquid (hunk 1 above)
Comment (ndouglas) [reply src/site/includes/vet_centers/address_phone_image.liquid:69]: The `vetCenterUseCapHours == false` specifically.

File: src/site/includes/vet_centers/address_phone_image.liquid (hunk 2 above)
Comment (maxx1128) [reply src/site/includes/vet_centers/address_phone_image.liquid:65]: I'm not too clear on this either. Do liquid files allow parenthesis in their conditionals to make these more explicit?

**Review by maxx1128 — CHANGES_REQUESTED** [review]

File: src/site/includes/vet_centers/address_phone_image.liquid (hunk 2 above)
Comment (ethanteague) [reply src/site/includes/vet_centers/address_phone_image.liquid]: Parenthesis are a no-go in liquid, and operators are read right to left. That said, this is confusing, so I broke this out into an additional elsif.

**Review by ndouglas — APPROVED** [review]

**Review by maxx1128 — APPROVED** [review]

**Pull request merged**